docs/release-notes/index.md

```diff
@@ -0,0 +1,114 @@
+---
+navigation_title: EDOT Python
+description: Release notes for Elastic Distribution of OpenTelemetry Python.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+---
+
+# Elastic Distribution of OpenTelemetry Python release notes [edot-python-release-notes]
+
+Review the changes, fixes, and more in each version of Elastic Distribution of OpenTelemetry Python.
+
+To check for security updates, go to [Security announcements for the Elastic stack](https://discuss.elastic.co/c/announcements/security-announcements/31).
+
+% Release notes include only features, enhancements, and fixes. Add breaking changes, deprecations, and known issues to the applicable release notes sections.
+
+% ## version.next [kibana-X.X.X-release-notes]
+
+% ### Features and enhancements [kibana-X.X.X-features-enhancements]
+% *
+
+% ### Fixes [kibana-X.X.X-fixes]
+% *
+
+## 1.3.0
+
+This release drops support for Python 3.8, following upstream changes.
```

Comment:
This is missing the link to upstream changelogs from the CHANGELOG

Reply:
Could you add it back? Can't find it atm